#!/usr/bin/env python
PACKAGE = "pipe_blaser_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("brightness_thresh", int_t, 0, "Lower threshold of laser brightness", 10, 0, 255)
gen.add("hue_thresh_1", int_t, 0, "Lower threshold of red hue, [th_1, 180]", 173, 160, 180)
gen.add("hue_thresh_2", int_t, 0, "Upper threshold of red hue, [0, th_2]", 7, 0, 20)
gen.add("sat_thresh", int_t, 0, "Saturation threshold [0, 255]", 100, 0, 255)

exit(gen.generate(PACKAGE, "pipe_blaser_ros", "LaserRingDetector"))